glean-core/ios/GleanTests/Debug/GleanDebugUtilityTests.swift

```diff
@@ -20,38 +20,37 @@ class GleanDebugUtilityTests: XCTestCase {
     }
 
     func testHandleCustomUrlLogPings() {
-        // Test initial state
-        XCTAssertFalse(Glean.shared.configuration!.logPings)
+        // We destroy the Glean handle so that Glean in in an unitialized state,
+        // this way it will save the value of `logPings` in the `logPings` prop.
+        Glean.shared.testDestroyGleanHandle()
 
         // Test toggle true
         var url = URL(string: "test://glean?logPings=true")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertTrue(Glean.shared.configuration!.logPings)
+        XCTAssertTrue(Glean.shared.logPings!)
 
         // Test invalid value doesn't cause setting to toggle
-        var previousValue = Glean.shared.configuration?.logPings
+        var previousValue = Glean.shared.logPings
         url = URL(string: "test://glean?logPings=Not-a-bool")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertEqual(previousValue, Glean.shared.configuration!.logPings)
+        XCTAssertEqual(previousValue, Glean.shared.logPings)
 
         // Test toggle false
         url = URL(string: "test://glean?logPings=false")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertFalse(Glean.shared.configuration!.logPings)
+        XCTAssertFalse(Glean.shared.logPings!)
 
         // Test invalid value doesn't cause setting to toggle
-        previousValue = Glean.shared.configuration?.logPings
+        previousValue = Glean.shared.logPings
         url = URL(string: "test://glean?logPings=Not-a-bool")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertEqual(previousValue, Glean.shared.configuration!.logPings)
-    }
+        XCTAssertEqual(previousValue, Glean.shared.logPings)
 
-    func testHandleCustomUrlWrongHost() {
         // This should NOT set the logPings to true or false because it doesn't
         // match the required host "glean".
-        let url = URL(string: "test://not-glean?logPings=true")
+        url = URL(string: "test://not-glean?logPings=true")
```

Comment:
I think you need to leave the `let` alone here, it is part of the variable definition of the `url`

Reply:
I can't, I made it all into one test (see https://github.com/mozilla/glean/pull/1058/files#diff-4f1b6d3f583d696ae3f4d66b64ae89d2L49). Leaving `let` gives me an error for trying to re-instantiate a variable with the same identifier as a previously created one.

Reply:
Oh, sorry, I overlooked that :)  Carry on!